source/deploy/try-mattermost.rst

```diff
@@ -0,0 +1,80 @@
+Try Mattermost
+===============
+
+This guide provides instructions for quickly trying out Mattermost using either Docker or Azure Marketplace. These options are ideal for testing and evaluation purposes as they allow you to quickly get a Mattermost instance up and running
+for exploration and testing.
+
+Deployment Options
+------------------
+
+.. tab:: Azure Marketplace
+
+    Mattermost is available as a pre-configured virtual machine image in the Azure Marketplace. This option provides a more production-like environment for testing.
```

Comment:
I'm a bit unclear on what "production-like" means in this context? If it's recommended for production then that's a great path for a quick eval that then can become a prod instance. If it's not recommended for production we should be more clear here IMO

Reply:
Updated in the edit, agreed.